### ISSUE TITLE
query GROUP BY only shows one series result

### ISSUE THREAD

**Issue by RPing** [RPing/influx-prompt#8]
After [dumping `NOAA water` data source](https://docs.influxdata.com/influxdb/v1.8/query_language/sample-data/#download-and-write-the-data-to-influxdb), then query
`SELECT MEAN("index") FROM "h2o_quality" GROUP BY location,randtag`, result is
```
name: h2o_quality
time                  mean               
---                   ---                
1970-01-01T00:00:00Z  50.69033760186263  

```
which is different from official influx cli:
```
name: h2o_quality
tags: location=coyote_creek, randtag=1
time	mean
----	----
0	50.69033760186263


name: h2o_quality
tags: location=coyote_creek, randtag=2
time	mean
----	----
0	49.661867544220485


name: h2o_quality
tags: location=coyote_creek, randtag=3
time	mean
----	----
0	49.360939907550076


name: h2o_quality
tags: location=santa_monica, randtag=1
time	mean
----	----
0	49.132712456344585


name: h2o_quality
tags: location=santa_monica, randtag=2
time	mean
----	----
0	50.2937984496124


name: h2o_quality
tags: location=santa_monica, randtag=3
time	mean
----	----
0	49.99919903884662

```
Either influxdb `1.6.4` or `1.8.0` shows the same result.

**Comment by 404cn** [comment]
I got this to but with ```show tag values with key = "some_key"

It only give me one measurments's tag value.

**Comment by RPing** [comment]
fixed this by #9 
@404cn If you encounter any issue on latest `0.1.2`, feel free to open a new issue and provide some details :)
